### PR TITLE
mail-mta/netqmail: Update mkservercert to use new path for functions.sh

### PR DESCRIPTION
**Pull request by ShyPixie**
@mgorny 

https://bugs.gentoo.org/show_bug.cgi?id=504116
https://bugs.gentoo.org/show_bug.cgi?id=504392

### CODE REVIEW

File: mail-mta/netqmail/netqmail-1.06-r4.ebuild [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+			else
+				epatch "${QMAIL_SPP_S}"/netqmail-spp.diff
+			fi
+			cd "${WORKDIR}"
+			epatch "${FILESDIR}"/genqmail-20080406-ldflags.patch
+			cd -
+		fi
+	fi
+
+	cd "${WORKDIR}"
```
Comment (mgorny) [inline mail-mta/netqmail/netqmail-1.06-r4.ebuild]: If you `cd`, then you need to 'cd back' (i.e. `cd -` above). Also, both need `|| die`. You don't have to fix the whole ebuild but please don't introduce more issues.

**Comment by mgorny** [comment]
As a side bonus, you may want to look into PGP-signing your commits. https://wiki.gentoo.org/wiki/Project:Gentoo-keys/Generating_GLEP_63_based_OpenPGP_keys is the usual doc I recommend.

**Comment by ShyPixie** [comment]
I fixed it. (and now the commit is signed.)

**Comment by kensington** [comment]
@vapier

File: mail-mta/netqmail/netqmail-1.06-r4.ebuild [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+	epatch "${DISTDIR}"/${QMAIL_LARGE_DNS}
+	epatch "${FILESDIR}"/${PV}-fbsd-utmpx.patch
+
+	ht_fix_file Makefile*
+
+	if ! use vanilla; then
+		# This patch contains relative paths and needs to be cleaned up.
+		sed 's~^--- ../../~--- ~g' \
+			<"${DISTDIR}"/${QMAIL_TLS_F} \
+			>"${T}"/${QMAIL_TLS_F}
```
Comment (hasufell) [inline mail-mta/netqmail/netqmail-1.06-r4.ebuild]: missing `|| die`

File: mail-mta/netqmail/netqmail-1.06-r4.ebuild [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+	cd "${WORKDIR}" || die
+	epatch "${FILESDIR}"/use-new-path-for-functions.sh.patch
+	cd - || die
+
+	qmail_src_postunpack
+
+	# Fix bug #33818 but for netqmail (Bug 137015)
+	if ! use authcram; then
+		einfo "Disabled CRAM_MD5 support"
+		sed -e 's,^#define CRAM_MD5$,/*&*/,' -i "${S}"/qmail-smtpd.c
```
Comment (hasufell) [inline mail-mta/netqmail/netqmail-1.06-r4.ebuild]: missing `|| die`

**Comment by ShyPixie** [comment]
@hasufell I fixed it.

**Comment by hasufell** [comment]
Ideally, the bugs would also be referenced in the commit message.

**Comment by ShyPixie** [comment]
@hasufell done.

**Comment by mgorny** [comment]
You missed the dependency on gentoo-functions but added it for you in extra commit, and merged.

**Pull request merged**